lib/rules/no-unsafe-innerhtml.js

```diff
@@ -27,6 +27,11 @@ module.exports = {
 
         // valid operators to match against, such as X.innerHTML += foo
         var OPERATORS = ["=", "+="];
+        // valid operators to not check, such as X.innerHTML *= 12; is likely very safe
+        // This list explicitly doesn't check ["=", "+="] or any newer operators that have not been reviewed
+        // - https://github.com/estree/estree/blob/master/es5.md#assignmentoperator
+        // - https://github.com/estree/estree/blob/master/es2016.md#assignmentoperator
+        var ALLOW_OPERATORS = ["-=", "*=", "/=", "%=", "<<=", ">>=", ">>>=", "|=", "^=", "&=", "**="];
```

Comment:
Even better: Let's use GOOD_ and BAD_. They are easier to distinguish when reading and it's shorter :)